e2e/framework/util.go

```diff
@@ -0,0 +1,20 @@
+package framework
+
+import "time"
+
+func Retry(attempts int, delay time.Duration, f func() error) error {
```

Comment:
I can switch to this. The Retry function here is just a copy of the pattern used all over the mantle (pluton/kola) code repo.